practices/refactoring/readme.md

```diff
@@ -0,0 +1,93 @@
+# Refactoring
+
+Refactoring is very tightly coupled to Simple Design and TDD. TDD enables simple design, while the practice of refactoring ensures that our code bases adapt to new or changing requirements while still keeping its maintainability.
+
+## The What and The Why
+
+### The What
+
+Refactoring is a systematic process for improving code without creating new functionality.
+
+### The Why
+
+The goal of refactoring is to remediate technical debt and transform sloppy code into [clean code](/corevalues/cleancode/cleancode.md) with a simple design.
```

Comment:
probably want to link the simple design here as well, though probably need that PR first

Reply:
Added it in. Hopefully it will link appropriately when the PR goes through.